pocs/tests/test_camera.py

```diff
@@ -222,7 +206,7 @@ def test_exposure(camera, tmpdir):
     # A one second normal exposure.
     camera.take_exposure(filename=fits_path)
     # By default take_exposure is non-blocking, need to give it some time to complete.
-    time.sleep(5)
+    time.sleep(10)
```

Comment:
Seems like a long timeout. Is this whole file skipped if camera is not detected?

Reply:
The long timeout turned out to be necessary for the FLI, which has a very long readout time. This test will run with the camera simulator even with no real cameras present, and the simulator needs less time. I should probably parameterise the timeout to avoid unnecessary delays.